File: Pension-Management-Frontend/Pension-Management-Portal/src/app/model/process-pension-input.ts
export class ProcessPensionInput {
    public aadhaarNumber: string;
	public pensionAmount: number;
	public bankServiceCharge: number;

    constructor() {
        this.aadhaarNumber = '';
        this.pensionAmount = 0;
        this.bankServiceCharge = 0;
    }
}

File: Pension-Management-Backend/Process-Pension-Microservice/src/main/java/com/cognizant/process/pension/model/ProcessPensionInput.java
package com.cognizant.process.pension.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ProcessPensionInput {
	private String aadhaarNumber;
	private Double pensionAmount;
	private Double bankServiceCharge;
}
